src/styles/main.scss

```diff
@@ -62,6 +66,7 @@
 // The secondary column is hidden on small screens
 .algolia-docsearch-suggestion--subcategory-column {
   display: none;
+  cursor: default;
```

Comment:
Tried to disable the click using `pointer-events: none;` but didn't work. I guess we'd need to disable  in the JS by checking if the parent of the click elements if different from the left column.

For now when a user click it redirect to the page of suggestion that is at the same level as the mouse. And not to the sub-category, like the user could expect.